rotkehlchen/tests/unit/test_ethereum_inquirer.py

```diff
@@ -304,3 +307,88 @@ def test_get_blocknumber_by_time_subgraph(ethereum_inquirer):
 def test_get_blocknumber_by_time_etherscan(ethereum_inquirer):
     """Queries etherscan for known block times"""
     _test_get_blocknumber_by_time(ethereum_inquirer, True)
+
+
+@pytest.mark.parametrize(*ETHEREUM_NODES_PARAMETERS_WITH_PRUNED_AND_NOT_ARCHIVED)
```

Comment:
for this and the other tests in here if they do remote connections, they should use vcr believe.